### PR TITLE
Accessibility 242

### PR DESCRIPTION
**Pull request by thegentlemanphysicist**
This is my first PR on the accessibility PR. #242 
**What it covers:**
- It ensures the headings in the page banners are h1's 
- It fixes the resizing issue with simple page banner
- inline with reducing tech debt it refactors the simple banners styling into styled-components
- redundant css has been removed
- Quick fix to Bug #332 we prevent the crash by checking that a case study exists before trying to load it.
(This came up while refactoring and I decided to fix it asap. It is a single line of code)

**What it does not cover:**
- tags withing cards
- tags within pages

### CODE REVIEW

File: react-frontend/src/components/StyleComponents/bannerSimple.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components';
+
+export const BannerSubTitle = styled.p.attrs({
+  className: 'bannerSubTitle',
+})`
+  color: #313132;
+  font-size: 22px;
+  position: relative;
+  @media only screen and (max-width: 800px) {
+    font-size: 10pt;
```
Comment (mark-a-wilson) [inline react-frontend/src/components/StyleComponents/bannerSimple.js:10]: Please remove `font-size: 10pt;`

File: react-frontend/src/components/StyleComponents/bannerSimple.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    text-align: justify;
+    font-size: 16px;
+    text-align: center;
+    width: 100%;
+  }
+`;
```
Comment (mark-a-wilson) [inline react-frontend/src/components/StyleComponents/bannerSimple.js:16]: for **`bannerSubTitle`**, to match @singhinderdeep 's Figma designs, I believe you can delete the `@media` query, and replace this block with:

```
   color: #313132;
   font-size: 22px;
   font-weight: 700;
   text-align: center;
```

File: react-frontend/src/css/pagetop.css [hunk 3]
```diff
@@ -36,6 +26,8 @@
 }
 
 .plainPageText {
+    display:block;
```
Comment (mark-a-wilson) [inline react-frontend/src/css/pagetop.css:29]: Is this class name being used anywhere now? (I could not find.) If not, delete?

**Review by mark-a-wilson — APPROVED** [review]
I've looked at the deployment and can verify this PR is fixing what it claims to fix. ✔️

And I can see that the code is getting more manageable and legible! 👏

File: react-frontend/src/css/pagetop.css (hunk 3 above)
Comment (thegentlemanphysicist) [reply react-frontend/src/css/pagetop.css]: good catch, I have deleted it.

**Pull request merged**